src/main/java/sirius/biz/analytics/explorer/EventTimeSeriesComputer.java

```diff
@@ -106,6 +106,7 @@ public EventTimeSeriesComputer(Class<E> eventType) {
      * @see #AGGREGATION_EXPRESSION_COUNT
      */
     public EventTimeSeriesComputer<O, E> addAggregation(String expression, @Nullable String label) {
+        // On first sight using a countIf would be way slower than a GROUPBY, but Clickhouse optimizes this so well that the query is way faster with a countIf
```

Comment:
I would make this part of the JavaDoc, not an inline comment.
Secondly I would describe it a bit deeper. Will post a text suggestion here soon...

Reply:
JavaDoc suggestion:
```java
    /**
     * Adds an aggregation expression to collect.
     * <p>
     * In likely most of the cases, the aggregations used with this method are conditional
     * count functions such as <code>countIf(<CLAUSE>)</code>. Performance measurements in Clickhouse
     * showed faster performance using such functions instead of regular <code>GROUP BY</code>
     * operators.
     *
     * @param expression the SQL expression to add to the <tt>SELECT</tt> clause
     * @param label      the label to use or <tt>null</tt> to use the default label
     * @return the computer itself for fluent method calls
     * @see #AGGREGATION_EXPRESSION_COUNT
     */
```